main.js

```diff
@@ -30,13 +36,19 @@ app.post("/", async (req, res) => {
 
   const chain = await getChain(res);
   await chain.call({
-    input_documents: await (await search_index).similaritySearch(templated, 4),
+    input_documents: await searchIndex.similaritySearch(templated, 4),
     question: templated,
   });
 
   res.end();
 });
 
-app.listen(port, () => {
+app.listen(port, async () => {
   console.log(`Started server on port: ${port}`);
+  console.log('initializing search index...');
```

Comment:
```suggestion
  console.log('Initializing search index...');
```

Reply:
We should also probably log when it has been initialized

Reply:
updated!